code/script.js

```diff
@@ -251,48 +255,73 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
+  questions.selectedIndex = defaultStatus
+  generateBoard()
+  setSecret()
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// Setting the currentQuestion object when you select something in the dropdown
+// ! It doesn't work with shorthands for some reason. I know I could use it here though !
```

Comment:
I haven't even tried the shorthand approach, so good job for trying! Maybe practice with an easier, less nested example so you know how the shorthand's working.